Importer/Event/Error/ExceptionEvent.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Netdudes\ImporterBundle\Importer\Event\Error;
+
+use Netdudes\ImporterBundle\Importer\Log\LogInterface;
+use Symfony\Component\EventDispatcher\Event;
+
+class ExceptionEvent extends Event
+{
+    /**
+     * @var LogInterface
+     */
+    protected $log;
+
+    /**
+     * @var \Exception
+     */
+    protected $exception;
+
+    /**
+     * @var bool
+     */
+    protected $stopped = false;
+
+    /**
+     * @var int|null
+     */
+    protected $index;
+
+    /**
+     * @param LogInterface $log
+     * @param \Exception   $exception
+     * @param int|null     $index
```

Comment:
So maybe we should have different types of events for these 2 cases - one with index, one without?
Because later, you have to do:

```
if (is_null($index)) {
    $log->addCriticalError('Interpreter exception events should always contain the index of the data.');

    return;
}
```

to handle this difference.